userspace/libscap/scap_bpf.c

```diff
@@ -1290,11 +1367,13 @@ static int32_t set_default_settings(scap_t *handle)
 {
 	struct scap_bpf_settings settings;
 
-	if(set_boot_time(handle, &settings.boot_time) != SCAP_SUCCESS)
+	uint64_t boot_time = 0;
+	if(set_boot_time(handle, &boot_time) != SCAP_SUCCESS)
```

Comment:
out of curiosity, how was this a warning?

Reply:
IIRC it was an `address-of-packed-member` warning.